modules/installation-localzone-generate-k8s-manifest.adoc

```diff
@@ -113,9 +113,8 @@ $ export AMI_ID=$(grep ami
 +
 [source,terminal]
 ----
-$ export SUBNET_ID=$(aws cloudformation describe-stacks \
-  --stack-name "<subnet_stack_name>" \ <1>
-  | jq -r .Stacks[0].Outputs[0].OutputValue)
+$ export SUBNET_ID=$(aws cloudformation describe-stacks --stack-name "<subnet_stack_name>" \
+  | jq -r '.Stacks[0].Outputs[0].OutputValue') <1>
```

Comment:
As per writing standards, the callout should be in the line where the intended field is listed.
```suggestion
$ export SUBNET_ID=$(aws cloudformation describe-stacks \
  --stack-name "<subnet_stack_name>" \ <1>
  | jq -r '.Stacks[0].Outputs[0].OutputValue') 
```

Reply:
Pasting the guideline here for reference:
```
If your command contains multiple lines and uses callout annotations, you must comment out the callout(s) in the codeblock, as shown in the following example:

To scale based on the percent of CPU utilization, create a `HorizontalPodAutoscaler` object for an existing object:

[source,terminal]
----
$ oc autoscale <object_type>/<name> \// (1)
  --min <number> \// (2)
  --max <number> \// (3)
  --cpu-percent=<percent> (4)
```

Reply:
Acknowledged.